config/default.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
what is this? Old config? 
Currently we work off of config.js which consumes environment variables.

Reply:
Old config, could be cleaned out